src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -30,14 +30,45 @@
 		</para>
 		<mediaobject>
 			<imageobject>
-				<imagedata fileref="monitor_01_login.jpg" format="JPG"
-					scalefit="1" />
+				<imagedata fileref="monitor_01_login.jpg" format="JPG" scalefit="1"/>
 			</imageobject>
 		</mediaobject>
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during
+				installation as<emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using
+				Apache Tomcat and its system service then this resources folder needs to be created manually as
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>. If you encounter any
+				problems while using the system service, make sure the resource folder is available.
+			</para>
+		</section>
+		<section>
+			<title>Manual creation of Windows systemprofile resource folder</title>
+			<para>
+				Default Windows explorer does not allow you to create a folder starting with a dot ('.')
```

Comment:
I think Windows Explorer is usually written with capital E. Maybe drop the "Default" then, since it's then clear you are talking about the built-in one.